Makefile

```diff
@@ -47,15 +49,15 @@ image-push: image
 	docker push ${IMAGE_PREFIX}${IMAGE_NAME}:${IMAGE_TAG}
 
 .PHONY: deploy
-deploy: manifests
-	kustomize build manifests/namespace-install | kubectl apply -f -
+deploy: kustomize manifests
```

Comment:
It looks like `manifests` depends on `kustomize` already so we don't need an explicit `kustomize` dependency here.